libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -152,7 +152,7 @@ namespace eosio { namespace testing {
 
          virtual ~base_tester() {};
 
-         void              init(const setup_policy policy = setup_policy::full, db_read_mode read_mode = db_read_mode::SPECULATIVE);
+         void              init(const setup_policy policy = setup_policy::full, db_read_mode read_mode = db_read_mode::SPECULATIVE, optional<uint32_t> genesis_max_inline_action_size = optional<uint32_t>{}, optional<uint32_t> genesis_max_nonprivileged_inline_action_size = optional<uint32_t>{});
```

Comment:
`genesis_max_nonprivileged_inline_action_size` this is really not genesis. It is startup config. I think it is confusing to call it genesis.